tools/test.py

```diff
@@ -147,7 +147,7 @@ def PrintFailureHeader(self, test):
     })
     print("Path: %s" % "/".join(test.path))
 
-  def Run(self, tasks):
+  def Run(self, tasks) -> dict:
```

Comment:
I believe that [lowercase `dict`](https://docs.python.org/3/whatsnew/3.9.html#type-hinting-generics-in-standard-collections) will only work on Py3.9 or greater so we need to ensure that these tests are only run in those conditions or:
1. `from typing import Dict` and then `-> Dict` __or__
2. Replace `from __future__ import print_function` with `from __future__ import annotations`

Reply:
> 1. `from typing import Dict` and then `-> Dict`

Thanks, I made this change.